src/Tizen.Telephony/Tizen.Telephony/Sim.cs

```diff
@@ -129,8 +129,14 @@ public enum ApplicationType
         /// Gets the Integrated Circuit Card IDentification (ICC-ID).
         /// The Integrated Circuit Card Identification number internationally identifies SIM cards.
         /// </summary>
+        /// <remarks>
+        /// For avoding the unexpected behavior of old version applications that have http://tizen.org/privilege/telephony privilege. There is an exceptional handling in case of permission denied.
```

Comment:
Spelling, should be `avoiding`. It is correct, but I suggest using `To avoid` instead.  It should be one sentence.

Reply:
Done